Content.Client/StatusIcon/StatusIconOverlay.cs

```diff
@@ -106,8 +106,8 @@ protected override void Draw(in OverlayDrawArgs args)
                         accOffsetR += texture.Height;
                         countR++;
                     }
-                    yOffset = (bounds.Height + sprite.Offset.Y) / 2f - (float)(accOffsetR - proto.Offset) / EyeManager.PixelsPerMeter;
-                    xOffset = (bounds.Width + sprite.Offset.X) / 2f - (float)texture.Width / EyeManager.PixelsPerMeter;
+                    yOffset = (bounds.Height + sprite.Offset.Y) / 2f - (float) (accOffsetR - proto.Offset) / EyeManager.PixelsPerMeter;
```

Comment:
wrong convention
we don't use a space behind a cast
check if your .editorconfig is being read correctly, your IDE should complain about this
```suggestion
                    yOffset = (bounds.Height + sprite.Offset.Y) / 2f - (float)(accOffsetR - proto.Offset) / EyeManager.PixelsPerMeter;
```
same for the other cases